packages/common/src/store/buy-crypto/sagas.ts

```diff
@@ -256,6 +257,17 @@ function* doBuyCryptoViaSol({
   const audiusBackendInstance = yield* getContext('audiusBackendInstance')
   const { track, make } = yield* getContext('analytics')
   const reportToSentry = yield* getContext('reportToSentry')
+  const audiusLocalStorage = yield* getContext('localStorage')
+
+  yield* call(
+    track,
+    make({
+      eventName: Name.BUY_CRYPTO_STARTED,
+      mint,
+      provider,
+      amount
```

Comment:
What unit is amount in? For the other analytics events, I think this get converted to dollars to make it easier to read in the analytics UI.

Reply:
it's in the UI format for the cryptocurrency requested (so dollars for USDC, SOL for SOL, AUDIO for AUDIO, never wei)